### PR TITLE
Add Feemanager call bypass in helper

### PR DESCRIPTION
**Pull request by ianrowan**
## Link to Issue
Closes: #7974

## Description of Changes
- Bypass additional balance check for oneOff contests using optional param

### CODE REVIEW

File: libs/model/src/services/commonProtocol/contestHelper.ts [hunk 1]
```diff
@@ -55,17 +55,28 @@ export const addContent = async (
   contest: string,
   creator: string,
   url: string,
+  web3?: Web3,
```
Comment (Rotorsoft) [inline libs/model/src/services/commonProtocol/contestHelper.ts:58]: are we exposing web3 again?

File: libs/model/src/services/commonProtocol/contestHelper.ts [hunk 2]
```diff
@@ -55,17 +55,28 @@ export const addContent = async (
...
+  nonce?: number,
```
Comment (Rotorsoft) [inline libs/model/src/services/commonProtocol/contestHelper.ts:59]: guess we are exposing the nonce implementation details after all 😆

File: libs/model/src/services/commonProtocol/contestHelper.ts (hunk 1 above)
Comment (rbennettcw) [reply libs/model/src/services/commonProtocol/contestHelper.ts:58]: yes and no– it's optional

File: libs/model/src/services/commonProtocol/contestHelper.ts (hunk 2 above)
Comment (rbennettcw) [reply libs/model/src/services/commonProtocol/contestHelper.ts:59]: FWIW, the batch funcs don't expose it– these old funcs can be treated as private/internal now

Comment (Rotorsoft) [reply libs/model/src/services/commonProtocol/contestHelper.ts:59]: just trying to follow the open-closed principle, but guess we can deal with the potential debt of opening these details when we start accumulating it in the future

Comment (rbennettcw) [reply libs/model/src/services/commonProtocol/contestHelper.ts:59]: agreed– but in the context of the batching, these args are actually needed to get the nonce + shared web3 instance from the batch caller

we just need to ensure that the single-contest funcs remain unexposed and cannot be used outside the helper lib.

Comment (Rotorsoft) [reply libs/model/src/services/commonProtocol/contestHelper.ts:59]: ok, so are we removing the export clause?

Comment (ianrowan) [reply libs/model/src/services/commonProtocol/contestHelper.ts:59]: These functions will likely be pushed in @rbennettcw work so feel free to open/close functions as desired based on what required. As mentioned if we're always calling batch version the old versions dont need to be exported but will allow existing work to refactor to ensure that doesnt break any other current work

**Comment by ianrowan** [comment]
These changes were cherry picked

**Pull request closed without merging**